crates/tabby/src/serve/completions/prompt.rs

```diff
@@ -361,31 +361,27 @@ mod tests {
     #[test]
     fn test_build_prefix_readable() {
         let snippets = vec![
-            "def snippet_1():\n    print(\"This is snipptet 1\")\n".to_string(),
-            "def snippet_2():\n    print(\"This is snipptet 2\")\n".to_string(),
-            "def snippet_3():\n    print(\"This is snipptet 3\")\n".to_string(),
-            "def snippet_4():\n    print(\"This is snipptet 4\")\n".to_string(),
-            "def snippet_5():\n    print(\"This is snipptet 5\")\n".to_string(),
+            "snippet_1() = build_snippet_1(n)".to_string(),
```

Comment:
please don't have word `snippet` present in test case string.

I recommend keywords like `invoke_function`

Reply:
Sure